### DISCUSSION TITLE
Possible unenrollment exploit for v120+?

### DISCUSSION THREAD

**Discussion by AnOrangeCone** [3kh0/ext-remover#1058] [Exploits JOIN TITANIUM NETWORK TO SHARE PLEASE]
When you attempt to enable dev mode with esc refresh power on the enrolling screen turn off your chromebook after like 5-8 seconds of being on that screen. Once it turns back on it will want you to sign in with an enrolled email. Is there a possible way to exploit this and possibly login by turning your internet off on this screen or something?

**Comment by FunnyCorgi** [comment]
Maybe? I've gotten mine to give me a google login, but I haven't tried signing in with a gmail yet. It might work.

So, what I see is that it has the enter email box, with the organization name at the end(probably device policy?) but I haven't tried adding a personal account yet. I have found that this will show up if you add a network, then get to the enrolling screen and turn off wifi as soon as it shows "Enterprise enrollment." Keep in mind that this happened on 118, I found this around 2 weeks(or more) ago, and I forgot to CAUB my home network, so it updated 2 days ago

**Reply by AnOrangeCone** [reply]
this works on v120 and i think higher ive been working with it it could be possible with a kind of skiovox type method of getting to settings and possibly adding an account through the settings account page
not tested since i have state testing and cant do anything stupid right now the only reason i found this is the power went out while reenrolling for testing

**Comment by ghost** [comment]
dunroll
